### ISSUE TITLE
Event handlers don't update correctly

### ISSUE THREAD

**Issue by BCrlrd** [Pathemeous/sem-group-4#165]
For some reason changing the keys is broken again. I tested this on friday and it worked fine. Now it just sets everything to random keys.

< bug

**Comment by Pathemeous** [comment]
Relevant:
#164

**Comment by Pathemeous** [comment]
```
public static void setPlayer1Right(int value) {
        completeKeyBindings.put(KeyBindHelper.PLAYER1_RIGHT_KEY, value);
        player1Input.setLeftInput(new InputKey(value));
    }
```

The issue is caused by the setters assigning the new key to the wrong attribute of the input (they all set to the left).

This is caused by me (who added these lines to the setters), but I will be completely changing these setters instead, since the completeKeyBindings list in Settings should not be there in the first place (evidently it causes confusion and is error prone, as shown in this example).

< bug, organisation

**Comment by BCrlrd** [comment]
git blame Damian

< bug

**Comment by Pathemeous** [comment]
Fixed with #183
